src/main/kotlin/it/pagopa/aca/client/IbansClient.kt

```diff
@@ -26,9 +26,11 @@ class IbansClient(@Autowired @Qualifier("ibansApiClient") private val client: Ib
                 logger.info(
                     "Querying api config to retrieve iban for creditorInstitutionCode: $creditorInstitutionCode, request id: $requestId"
                 )
-                client.getCreditorInstitutionsIbansEnhanced(
+                client.getIbans(
+                    0,
                     creditorInstitutionCode,
                     requestId,
+                    5,
                     CREDITOR_INSTITUTION_LABEL
```

Comment:
We select the five ibans of the first page?

Reply:
is always used only the first `iban`, but it could be refactored, I'll try it